flask_rebar/validation.py

```diff
@@ -115,7 +115,13 @@ def disallow_extra_fields(self, processed_data, original_data):
     RequestSchema = Schema
 
 
+# Define a constructor to please some IDE type checking
 class ResponseSchema(ActuallyRequireOnDumpMixin, Schema):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
```

Comment:
Function call overhead in python is significant. It’s best practice not to introduce this extra no-op function call that reduces performance just for some IDEs that aren’t currently finding the inherited method. Can you follow up with your IDE vendor instead of adding this here?